### ISSUE TITLE
Installs wrong architecture on Raspberry Pi

### ISSUE THREAD

**Issue by webmink** [YunoHost-Apps/lemmy_ynh#54]
### Describe the bug

A valid fresh install on Raspberry Pi installs the x86 binary

### Context

- Hardware:  Raspberry Pi at home 
- YunoHost version: 11.1.21.4
- I have access to my server: *Through SSH | through the webadmin | direct access via keyboard / screen 
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no
- Using, or trying to install package version/branch:  0.16.7~ynh3

### Steps to reproduce

This was a fresh install from the web UI on a long-term-stable Yunohost instance. Log is below.

### Expected behavior

Listing page says all architectures are supported so I expected the ARM version to be installed.

### Logs

Service log: https://paste.yunohost.org/leverurugo

Install log:

Installing lemmy...
Provisionning sources...
Provisionning ports...
Provisionning system_user...
Provisionning install_dir...
Provisionning data_dir...
Provisionning permissions...
Provisionning apt...
Provisionning database...
Installing dependencies...
Creating a PostgreSQL database...
Setting up source files...
Configuring NGINX web server...
Making install...
Adding a configuration file...
Configuring a systemd service...
Configuring log rotation...
Integrating service in YunoHost...
Starting a systemd service...
(this may take some time)
The service lemmy didn't fully executed the action start before the timeout.
Please find here an extract of the end of the log of the service lemmy:
Jul 01 09:48:35 systemd[1]: Started Lemmy: Link aggregator.
Jul 01 09:48:35 systemd[1064808]: lemmy.service: Failed to execute /var/www/lemmy/lemmy/lemmy: Exec format error
Jul 01 09:48:35 systemd[1064808]: lemmy.service: Failed at step EXEC spawning /var/www/lemmy/lemmy/lemmy: Exec format error
Jul 01 09:48:35 systemd[1]: lemmy.service: Main process exited, code=exited, status=203/EXEC
Jul 01 09:48:35 systemd[1]: lemmy.service: Failed with result 'exit-code'.
\-\-\-
The service lemmy-ui has correctly executed the action start.
Installation of lemmy completed
Installation completed

**Comment by ericgaspar** [comment]
fixed with #55

**Comment by MohamedElashri** [comment]
The app says that it cannot be installed on arm64, even the `Testing` branch 

```
The application cannot be installed
This app can only be installed on specific architectures (amd64) but your server architecture is arm64.
```
